### PR TITLE
I changed some file structure, created edit type field and completed …

### PR DESCRIPTION
**Pull request by marat-sahakyan-923**
…work on delete field.

### CODE REVIEW

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  const dispatch = useDispatch();
+
+  const { data, toggleDeletedDialog } = useSelector(
+    (store) => store.tableDataInfo
+  );
+
+  const onCloseBtn = () => {
+    dispatch(changeShowView(false));
+    dispatch(changeShowCreateTabPopover(false));
+    dispatch(changeToggleDeletedProperties());
```
Comment (M7Arman) [inline src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx:32]: սրանք հաստատ պետք ա redux-ում լինեին՞

File: src/components/popovers/createEditTabpopovers/CreateTabPopoverJsx.jsx [hunk 2]
```diff
@@ -10,8 +10,6 @@ import {
   changeCreatedTabName,
```
Comment (M7Arman) [inline src/components/popovers/createEditTabpopovers/CreateTabPopoverJsx.jsx:10]: file name souldn't contain jsx word

File: src/components/deletedDialog/DeletedDialog.jsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+// import DialogActions from "@mui/material/DialogActions";
+import DialogContent from "@mui/material/DialogContent";
+// import DialogContentText from "@mui/material/DialogContentText";
+import DialogTitle from "@mui/material/DialogTitle";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import style from "./deletedDialog.module.scss";
+import { changeToggleDeletedDialog } from "../../features/tableDataInfo/tableDataInfoSlice";
+
+export default function DeletedDialog({ type, onDelete, id }) {
```
Comment (M7Arman) [inline src/components/deletedDialog/DeletedDialog.jsx]: DeletedDialog խի անցյալով՞

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  };
+
+  const unDeleteProperty = (id) => {
+    dispatch(changeSelectedPropertyForEdit(id));
+    dispatch(changeToggleDeletedProperties());
+    dispatch(changeToggleAddPropertyPopover(true));
+  };
```
Comment (M7Arman) [inline src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx:39]: are you sure that you need all these global states?

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx (hunk 1 above)
Comment (marat-sahakyan-923) [reply src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx:32]: yes I use them in global

File: src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx (hunk 4 above)
Comment (marat-sahakyan-923) [reply src/components/popovers/createEditTabpopovers/drawSelectedButton/properties/DeletedProperties.jsx:39]: yes I use them in global

**Pull request merged**